docker/Dockerfile

```diff
@@ -61,4 +126,8 @@ ENV XLA_FLAGS="--xla_gpu_enable_triton_gemm=false"
 ENV XLA_PYTHON_CLIENT_PREALLOCATE=true
 ENV XLA_CLIENT_MEM_FRACTION=0.95
 
-CMD ["python3", "run_alphafold.py"]
+ENV PATH="/hmmer/bin:/alphafold3_venv/bin:${PATH}"
+
+WORKDIR /app/alphafold
+
+CMD ["python3", "run_alphafold.py", "--help"]
```

Comment:
Ideally this would be split into a separate `ENTRYPOINT` and `CMD` sections, e.g.

```
ENTRYPOINT ["python3", "run_alphafold.py"]

CMD ["--help"]
```

It would then be possible to pass the arguments directly to the Docker command, e.g.

```
docker run -it alphafold3 --output_dir=/root/af_output
```

However, this would involve rewriting the documentation and is arguably less intuitive so I'm broadly replicating the current functionality.